documentation/book/proc-metrics-deploying-prometheus.adoc

```diff
@@ -0,0 +1,62 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-prometheus.adoc
+[id='proc-metrics-deploying-prometheus-{context}']
+
+= Deploying Prometheus
+
+To deploy Prometheus to your Kafka cluster, apply the link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml[Prometheus YAML^] resource file and the YAML files for Prometheus-related resources using the source for the Prometheus docker image in the `./metrics/examples/prometheus/install` directory of the repository.
+
+The deployment process creates a `ClusterRoleBinding` and discovers an Alertmanager instance in the namespace defined in the YAML.
+
+.Prerequisites
+
+* Check the xref:ref-metrics-alertmanager-examples-{context}[example alerting rules provided]
+
+.Procedure
+
+. Download and edit the Prometheus resource file from the repository to use a different `namespace`:
++
+On Linux, use:
++
+[source,shell,subs="+quotes,attributes"]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
++
+On MacOS, use:
++
+[source,shell,subs="+quotes,attributes"]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+
+. Download and edit the `ServiceMonitor` resource in `strimzi-service-monitor.yaml` to define Prometheus jobs that will scrape the metrics data:
++
+[source,shell,subs="+quotes,attributes"]
+curl -O https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/strimzi-service-monitor.yaml
+
+. Currently, the Prometheus Operator only supports jobs that include an `endpoints` role for service discovery. To use another role:
```

Comment:
Our documentation doesn't really explain enough for a reader to understand what this means.

Reply:
I've expanded this in a note in the procedure introduction: 
NOTE: Currently, the Prometheus Operator only supports jobs that include an `endpoints` role for service discovery. Targets are discovered and scraped for each endpoint port address. For endpoint discovery, the port address may be derived from service (`role: service`) or pod (`role: pod`) discovery.

@ppatierno and @stanlyDoge - Does this mean we should only configure the Operator for endpiunt discovery? In the example `prometheus-additional.yaml` it's configured for node discovery.

Reply:
@ppatierno and @stanlyDoge - Does this mean we should only configure the Operator for endpiunt discovery? In the example prometheus-additional.yaml it's configured for node discovery.

Reply:
Changed the note to say 'By default,  ...'